src/apps/api/tests/test_competitions.py

```diff
@@ -69,3 +70,47 @@ def test_adding_organizer_accepts_them_if_they_were_existing_participant(self):
             competition=self.comp,
             status=CompetitionParticipant.APPROVED
         ).count() == 1
+
+
+class PhaseMigrationTests(APITestCase):
+    def setUp(self):
+        self.creator = UserFactory(username='creator', password='creator')
+        self.other_user = UserFactory(username='other_user', password='other')
+        self.comp = CompetitionFactory(created_by=self.creator)
+        self.phase_1 = PhaseFactory(competition=self.comp, index=0)
+        self.phase_2 = PhaseFactory(competition=self.comp, index=1)
+        self.leaderboard = LeaderboardFactory(competition=self.comp)
+        ColumnFactory(leaderboard=self.leaderboard)
+
+    def test_automigration_checks_permissions_must_be_collaborator_to_migrate(self):
+        self.client.login(username='other_user', password='other')
+
+        url = reverse('phases-manually_migrate', kwargs={"pk": self.phase_1.pk})
+        resp = self.client.post(url)
+        assert resp.status_code == 403
+        assert resp.data["detail"] == "You do not administrative permissions for this competition"
```

Comment:
spelling error 
```suggestion
        assert resp.data["detail"] == "You do not have administrative permissions for this competition"
```
probably means a spelling error where we are raising this error too, yea?

Reply:
ty! ✔️